### DISCUSSION TITLE
How to test infolist displayed data

### DISCUSSION THREAD

**Discussion by fabio-ivona** [filamentphp/filament#14458] [Help]
### Package

Infolist builder

### Package Version

v3.2.115

### How can we help you?

Hi All!

is there any way to test if infolists display correct data? something similar to `assertFormSet()` for forms

I have a view page with some `TextEntry` and I need to be sure that the displayed data matches expected

**Comment by orebarranco** [comment]
I have this test for infolists

```
it('can retrieve data for view page', function () {
    $user = User::factory()->create();

    livewire(name: ViewUser::class, params: ['record' => $user->getRouteKey()])
        ->assertSee($user->name)
        ->assertSee($user->email)
        ->assertSee($user->username);
});
```

**Reply by fabio-ivona** [reply]
That's the way I do that too, was hoping there was a way to test it like we do for inputs and columns

btw thanks for confirming this @orebarranco !
